src/FileFormats.Minidump/MinidumpStructures.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FileFormats.Minidump
+{
+    public static class CrashDumpLayoutManagerExtensions
+    {
+        public static LayoutManager AddCrashDumpTypes(this LayoutManager layouts, bool isBigEndian, bool is64Bit)
+        {
+            return layouts
+                     .AddPrimitives(isBigEndian)
+                     .AddEnumTypes()
+                     .AddSizeT(is64Bit ? 8 : 4)
+                     .AddPointerTypes()
+                     .AddNullTerminatedString()
+                     .AddTStructTypes();
+        }
+    }
+
+#pragma warning disable 0649
+#pragma warning disable 0169
+
+    internal class MINIDUMP_HEADER : TStruct
```

Comment:
I try to stick with BCL casing conventions that would name this MinidumpHeader (though to be fair I am not sure if I even consistently did that in my own code)